hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Server.java

```diff
@@ -3192,23 +3198,47 @@ protected Server(String bindAddress, int port,
     if (queueSizePerHandler != -1) {
       this.maxQueueSize = handlerCount * queueSizePerHandler;
     } else {
-      this.maxQueueSize = handlerCount * conf.getInt(
-          CommonConfigurationKeys.IPC_SERVER_HANDLER_QUEUE_SIZE_KEY,
-          CommonConfigurationKeys.IPC_SERVER_HANDLER_QUEUE_SIZE_DEFAULT);      
+      String handlerQueueSizeStr = conf.get(getQueueClassPrefix() + "." +
```

Comment:
We can also use ```conf.getInt``` here and as well as in the next few places?

Reply:
Thanks @tomscut  for the suggestion.